src/test/java/tech/toparvion/analog/util/config/ChoicesCustomConfigurationLoaderTest.java

```diff
@@ -0,0 +1,102 @@
+package tech.toparvion.analog.util.config;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.boot.SpringApplication;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.MutablePropertySources;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+class ChoicesCustomConfigurationLoaderTest {
+  private static final String PATH_PREFIX = "src/test/resources/autoReload/";
+
+  @Mock
+  private ConfigurableEnvironment environment;
+  @Mock
+  private SpringApplication springApplication;
+  @Mock
+  private MutablePropertySources mutablePropertySources;
+  @InjectMocks
+  private ChoicesCustomConfigurationLoader sut;
+
+  @BeforeEach
+  void setUp() {
+    initMocks(this);
+
+    when(environment.getPropertySources()).thenReturn(mutablePropertySources);
+  }
+
+  @AfterEach
+  void afterEach() {
+    verifyNoInteractions(springApplication);
```

Comment:
What is this verification for?

Reply:
``ChoicesCustomConfigurationLoader`` implements ``EnvironmentPostProcessor``-interface and method ``postProcessEnvironment``, declared in this interface, contains ``SpringApplication`` as a parameter. 

There are so many scared things whitch somebody might do with ``SpringApplication`` 😄 
This verification is some kind of prove, that ``ChoicesCustomConfigurationLoader`` don't touch ``SpringApplication`` and, as a result, don`t do one of these scared things.

### **Example**
* Let's add just one line to this method
```java
  @Override
  public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
    application.run("");
    ......
```
* Start AnaLog
* Result will be so sad:
```
15:38:00.683 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
java.lang.StackOverflowError: null
```

Reply:
Great! No more questions here.